src/FileType_Fortran.f90

```diff
@@ -857,6 +861,28 @@ SUBROUTINE setStatus_fortran_file(file,status)
       ENDIF
     ENDSUBROUTINE setStatus_fortran_file
 !
+!------------------------------------------------------------------------------
+!> @brief This subroutine writes a 1-D array of strings to an output file as
+!>        lines.
+!> @param file The fortran file where the lines are written.
+!> @param lines The 1-D array of strings to write.
+!>
+    SUBROUTINE write_str_1a_fortran_file(file,lines)
+      CHARACTER(LEN=*),PARAMETER :: myName='writeTable_fortran_file'
+      CLASS(FortranFileType),INTENT(INOUT) :: file
+      TYPE(StringType),INTENT(IN) :: lines(:)
+      INTEGER(SIK) :: i
+
+      IF(file%initstat) THEN
+        IF(file%isOpen()) THEN
```

Comment:
Should probably make both of these IF checks DBC?  We shouldn't write to an unopened or nonexistent file

Reply:
Makes sense.  Do we want it to call `%fopen` if it isn't open?  How do we do it in the HDF5 file?

Reply:
Looks like we throw an error if the file isn't opened before the call.  I can make both DBC checks.

Reply:
I think the HDF5 requires you to open it before calling read/write, but I see no harm in opening the first here if it's not yet.  Just make sure to close it after writing if it was closed upon entering the routine so that the open/closed status of the file is unchanged upon return.

I would kind of like to treat the hdf5 file the same way and delete a LOT of open/close statements through MPACT, but that's a separate discussion